scripts/setup-web2py-debian.sh

```diff
@@ -25,17 +25,20 @@ read CONFIRM
 echo "installing useful packages"
 echo "=========================="
 apt-get update
+apt-get -y install sudo
```

Comment:
it thrown an error when execute in docker container image because of minimal installed base system, think there is another option to cover the error, either get the user to installed sudo package first before execute the install script or put the sudo package on install scripts like the proposed change script
ref:
https://groups.google.com/forum/#!topic/web2py/IFasLOdWsbI